twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -379,7 +372,7 @@ def _scrooge_scala_library_impl(ctx):
     )
     if ctx.attr.exports:
         exports = [exp[JavaInfo] for exp in ctx.attr.exports]
-        all_java = java_common.merge(exports + [aspect_info.java_info])
+        all_java = java_common.merge(_concat_lists(exports, [aspect_info.java_info]))
```

Comment:
nit: why not do `exports.append(aspect_info.java_info)`? that is O(1) in python so I hope skylark, whereas this is O(N) full copy.

Reply:
@iirina any thoughts? I’ll do a follow up OR and can fix this if you agree